testdata/config.toml

```diff
@@ -10,4 +10,14 @@ ipapico_enable = false
 ipapicom_enable = false
 
 [ips_config]
-
+peer_file_path = "testdata/peers.json"
+pref_location = true
+pref_location_closer = true
+pref_location_distance = 1000
+change_at_least = 1
+change_no_more = 2
+location_weight = 0.10
+degree_weight = 0.25
+eigenvector_weight = 0.25
+betweenness_weight = 0.25
+closeness_weight = 0.15
```

Comment:
Should we add a check to expect the sum of these is 1.0 in the code?

Reply:
That could not be truth. 
Eg. some users may want to define weights as 1.0, 1.0, 1.0 and 2.0 to emphasize that some factor is two times important than other and that also be fine. 
Some other users may want to define negative weights to give a penalty for some factor. And that also should be fine.